www/docs/en/dev/cordova/events/events.md

```diff
@@ -323,6 +323,16 @@ function onBackKeyDown() {
 }
 ```
 
+### Windows Quirks
+
+Throw an error in a `backbutton` callback to force the default behavior:
+
+```javascript
+document.addEventListener('backbutton', function (evt) {
+    throw "Exit"; // This will suspend the app
```

Comment:
The throw value is not used anywhere, it's just being caught in cordova-js and causes the event handler to indicate that the event has not been handled:
https://github.com/apache/cordova-windows/blob/master/template/www/cordova.js#L1489